python/tests/rtl/test_api_settings.py

```diff
@@ -138,26 +142,25 @@ def test_credentials_from_env_variables_override_config_file(
     monkeypatch.setenv("LOOKER_BASE_URL", "https://host1.looker.com:19999")
     monkeypatch.setenv("LOOKER_CLIENT_ID", "id123")
```

Comment:
I'm not clear on how the environment variable credentials are tested any more

Reply:
@josephaxisa yeah, we should remove `LOOKER_CLIENT_ID` and `LOOKER_CLIENT_SECRET` from these tests and do some testing in `test_user_session.py` (which should be renamed to `test_auth_session.py`) to confirm the behavior when those settings are present via config file, env variable, or absent.